blackbox/docs/sql/functions_operators.txt

```diff
@@ -0,0 +1,21 @@
+.. highlight:: psql
+.. _sql_functions_operators:
+
+=======================
+Functions and Operators
+=======================
+
+This chapter provides an overview of functions and operators that are permitted
+for writing expressions in CrateDB. Operators are reserved words that are
+primarily used in a WHERE clause to perform arithmetic operations or

```

Comment:
I wouldn't refer to the WHERE clause... they can be used everywhere: SELECT, HAVING, ORDER BY, GROUP BY...